### DISCUSSION TITLE
StartBringIntoView on ListView with virtual items

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#13791] [Q&A]
I have a `ListView` with about 30 items, more than what fits the screen. From my understanding the items are virtualized, so not all the containers are created initially. My issue is that I want to scroll the listview to the bottom and bring that item into view. There is a method on the `ListViewItem` called `StartBringIntoView`, which works great for items that already have a ListViewItem container, but I can't figure out how to make the list view go to the last item (or any "virtual" item that isn't created yet).
_Edit: `ScrollIntoView` doesn't work, see discussion below_

**Comment by jeromelaban** [comment]
Thanks for the question, you're looking for [`ScrollIntoView`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.scrollintoview?view=winrt-22621).

**Reply by HakanL** [reply]
Ah, I forgot to mention that. Calling `ScrollIntoView` didn't do anything. I called it with the last item (tested with any item) in the ItemsSource.

**Reply by jeromelaban** [reply]
That would be a good issue to open with more details and a sample of what does not work. Thanks!

**Reply by HakanL** [reply]
As I tested it in a standalone project, I think I got the answer:
fail: Microsoft.UI.Xaml.Controls.ListViewBase[0] The member void ListViewBase.ScrollIntoView(object item) is not implemented. For more information, visit https://aka.platform.uno/notimplemented?m=Microsoft.UI.Xaml.Controls.ListViewBase.void%20ListViewBase.ScrollIntoView%28object%20item%29

That's with 4.10.22 of Uno.

**Reply by HakanL** [reply]
The sample project is here: https://github.com/HakanL/Uno-ListView-Scroll-Issue

**Reply by jeromelaban** [reply]
Thanks, indeed this is not yet supported: https://github.com/unoplatform/uno/issues/4772

**Reply by HakanL** [reply]
Ah I see, and I assume there is no other way to have it change the scroll position? Short of changing it to ItemsRepeater.
